server/db_mysql.go

```diff
@@ -195,6 +196,39 @@ func (ms *MsqlCache) GetFixity(start string, end string, item string, status str
 	return results
 }
 
+func (ms *MsqlCache) PostFixity(item string) (int, error)  {
+        var err error
+        fixity_results := ms.GetFixity("*", "*", item, "scheduled")
+        if len(fixity_results) > 0 {
+                err = ms.DeleteFixity(fixity_results[0].Id)
+                if err != nil {
+                        return 500, err
+                }
+        }
+
+        err = ms.UpdateFixity(item, "scheduled", "")
```

Comment:
I like that this is reusing other methods to do the SQL call.
If this logic doesn't need any actual SQL statements, maybe it could be moved out of the database specific file and put directly in the HTTP handler or some place similar.